### ISSUE TITLE
Pre-train DAMSM

### ISSUE THREAD

**Issue by vawawo9029** [huiyegit/T2I_CL#22]
How to decide which epoch has good performance to choose when training DAMSM + CL for 600 epochs ?

**Comment by vawawo9029** [comment]
There are word loss, sentence loss and valid loss when training DAMSM + CL ?
